analysis_service/base/forms.py

```diff
@@ -0,0 +1,448 @@
+import logging
+from django import forms
+from django.core.exceptions import ValidationError
+from django.template.defaultfilters import filesizeformat
+from analysis_service.base import models
+
+
+logger = logging.getLogger("django")
```

Comment:
I see the logger is not used anywhere in this module, can you please remove? Also, if you want to use it it's better to initialize it using 
```python
logger = logging.getLogger(__name__)
```
See [the python docs](https://docs.python.org/2/library/logging.html#logger-objects) for more info.